builder/management/commands/flush_huey.py

```diff
@@ -0,0 +1,20 @@
+from django.core.management.base import BaseCommand, CommandError
+from huey.contrib.djhuey import HUEY
+
+
+class Command(BaseCommand):
+    args = "[course key]"
```

Comment:
Was this file copied from somewhere? I think the `args` class variable is a relic from very old Django versions. The `add_arguments()` method has been used instead since Django v1.8 or so.

If this was copied, it would be best to add a link to the source in the module docstring.

Reply:
I had just copied the files from the access/management folder. Removed the extra field